tools/autograd/gen_variable_type.py

```diff
@@ -339,27 +341,42 @@
 
 # Generate a file that lists all functions and their schema string. Used for XLA
 REGISTRATION_DECLARATION = CodeTemplate("""\
-${return_type} ${api_name}(${formals}); // {"schema": "${schema_string}", "compound": "${compound}"}
+${return_type} ${api_name}(${declaration_formals}); // {"schema": "${schema_string}", "compound": "${compound}"}
 """)
 
 # ProfiledType templates
-PROFILE_DISPATCH_UNBOXED = CodeTemplate("""\
+UNBOXED_PROFILE_DISPATCH = CodeTemplate("""\
```

Comment:
Oops, didn't reach this duplication and the one below during the review yesterday. Sorry to pile on the nits but I really think it helps maintainability to factor this kind of stuff